models/Meme.js

```diff
@@ -4,7 +4,9 @@ module.exports = (sequelize, DataTypes) => {
         link: {
             type: DataTypes.STRING,
             validate: {
-                isUrl: true
+                isUrl: {
+                    msg: 'Your URL doesn\'t look correct'
```

Comment:
typo. 
Also, should the message be more informative? So either: your URL cannot be found  or  something something is not an URL. 
URL cannot be found versus it's not the correct format are different things, so I think a more specific message/differently phrased message would be better.

Reply:
I've made the error message a bit more descriptive. "Your URL doesn't have a proper structure"